src/backend/gporca/libgpopt/src/operators/CLogicalReturning.cpp

```diff
@@ -0,0 +1,204 @@
+//---------------------------------------------------------------------------
+//	Greenplum Database
+//	Copyright (C) 2013 EMC Corp.
```

Comment:
Please use follofing copyright header for now:
```suggestion
//	Copyright (c) 2025 Greengage Community
```

Reply:
done